scripts/fixupModules.cjs

```diff
@@ -7,6 +7,7 @@ const process = require('node:process');
 
 // incorrect packages without type specifier
 [
+  '@fragnova/api-augment',
```

Comment:
Is there an actual issue logged to fix this? If so please do add it like the one below.

(These will be removed within the next 7-ish days, we don't want to continue working around packages that don't quite specify export maps correctly. It is terrible having to fixup as part of the build.)

Reply:
@jacogr Sure, I have created the issue and updated this file with the issue URL. Thanks